styles.css

```diff
@@ -0,0 +1,140 @@
+body {
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    color: #271D19;
+    background: #F6F1EF;
+}
+
+a {
+    text-decoration: none;
+    color: white;
+  }
+  a:hover {
+    color: #94563D;;
+  }
+  
+  .main-nav {
+    width: 45%;
+    background: #EC784A;
+    padding: 5px;
+    position: absolute;
+    right: 0;
+    top: 0;
+    text-align: center;
+    border-top-left-radius: 5px;
+    border-bottom-left-radius: 5px;
```

Comment:
Nice!  I like the rounding on the left side, without rounding on the right side.